widgetssdk/src/main/res/values/strings.xml

```diff
@@ -70,20 +70,20 @@
 
     <!-- End screen sharing Dialog -->
     <string name="glia_dialog_screen_sharing_end_title">End Screen Sharing?</string>
-    <string name="glia_dialog_screen_sharing_end_message">Are You sure You want to end the screen sharing?</string>
+    <string name="glia_dialog_screen_sharing_end_message">Are you sure you want to end screen sharing?</string>
     <string name="glia_dialog_screen_sharing_end_cancel">@string/glia_dialog_cancel</string>
     <string name="glia_dialog_screen_sharing_end_end">End</string>
 
     <!-- Operators unavailable AlertDialog -->
-    <string name="glia_dialog_operators_unavailable_title">We\'re sorry</string>
+    <string name="glia_dialog_operators_unavailable_title">Operators Not Available</string>
```

Comment:
Do we need to have `Not` and `Available` uppercased?

Reply:
Updated according to doc that was provided please see under task